client/my-sites/checkout/composite-checkout/lib/submit-wpcom-transaction.ts

```diff
@@ -25,11 +25,12 @@ export default async function submitWpcomTransaction(
 	payload: WPCOMTransactionEndpointRequestPayload,
 	transactionOptions: PaymentProcessorOptions
 ): Promise< WPCOMTransactionEndpointResponse > {
-	const isJetpackUserLessCheckout =
-		payload.cart.products.some( ( product ) => product.extra.isJetpackCheckout ) &&
-		payload.cart.cart_key === 'no-user';
+	const isUserLessCheckout =
+		payload.cart.products.some( ( product ) => {
+			return product.extra.isJetpackCheckout || product.extra.isAkismetSitelessCheckout;
+		} ) && payload.cart.cart_key === 'no-user';
```

Comment:
The change here is a little bit similar to the one in the file above, but I decided not to make a helper function because this is very specific and might vary from other places where we would want only one check or the other (i.e. if it has the product.extra properties, or the cart_key, or even just one of the properties).